packages/@okta/vuepress-site/code/react/okta_react/index.md

```diff
@@ -385,25 +389,24 @@ const AppWithRouterAccess = () => {
   };
 
   return (
-    <Security issuer='https://${yourOktaDomain}/oauth2/default'
+    <Security issuer='https://{yourOktaDomain}/oauth2/default'
```

Comment:
I think the routes should be wrapped in a `<Switch>` component. Otherwise `/login` may render on `/login/callback`. Probably /login/callback needs to come before /login for the same reason

Reply:
I tried moving `/login/callback` first and it doesn't help. Do you have an example of using `<Switch>`? Maybe I should just use `/callback`? :trollface: